source/user-manual/wazuh-dashboard/troubleshooting.rst

```diff
@@ -250,8 +250,6 @@ Check the following log files:
 
           # journalctl -u wazuh-dashboard
 
-          # cat /usr/share/wazuh-dashboard/data/wazuh/logs/wazuhapp.log | grep -i -E "error|warn"
```

Comment:
Would it be useful a `cat WAZUH_PATH/logs/ossec.log` command grepping relevant dashboard errors rather than removing the outdated line?

Reply:
I think so, that command is adding a grep to the existent `journalctl -u wazuh-dashboard` command. I will add it the command.

Reply:
Fixed on 8755d257ce2f0b0958271eec9c9ebf427080be79